env.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Thks I will reflect your advice.

sorry for `conf.py`
I will commit as soon as.